packages/nimble-tokens/package.json

```diff
@@ -29,15 +29,15 @@
   },
   "homepage": "https://github.com/ni/nimble#readme",
   "devDependencies": {
-    "@ni/eslint-config-javascript": "^3.1.0",
+    "@ni/eslint-config-javascript": "file:../ni-eslint-config-javascript-3.1.0.tgz",
     "cross-env": "^7.0.3",
     "glob": "^8.0.1",
     "hex-rgb": "4.3.0",
     "lodash": "^4.17.21",
     "style-dictionary": "3.7.0",
     "svg-to-ico": "1.0.8",
     "svg-to-ts": "^6.0.1",
-    "typescript": "^4.3.2"
+    "typescript": "~4.3.2"
```

Comment:
We should try to keep the versions aligned if possible, the scripts in here are simple so I'd expect this can be upgraded to "typescript": "~4.6.4" like elsewhere (the tsconfig could also align the target and module versions to "es2020")